README.md

```diff
@@ -159,8 +159,9 @@ the "generic" system, a new empty Spack environment will be automatically create
 different Spack environment by setting the environment variable `EXCALIBUR_SPACK_ENV`
 described above.
 
-## Contributing new systems or benchmarks
+## Contributing new systems or benchmarks and other information
 
 Feel free to add new benchmark apps or support new systems that are part of the
 ExCALIBUR benchmarking collaboration.  Read
 [`CONTRIBUTING.md`](./CONTRIBUTING.md) for more details.
+The file [`SYSTEMS.md`](./SYSTEMS.md) contains information about the use of this framework on specific systems.
```

Comment:
I think it would make more sense to have this as a subsection under `Usage` since it's closely related. I'd also add a link at the end of "Selecting system and queue access options"

```suggestion
### System-specific flags

While the aim is to automate as much system-specific configuration as possible, there are some options that have to be provided by the user, such as accounting details, and unfortunately the syntax can vary. The file [`SYSTEMS.md`](./SYSTEMS.md) contains information about the use of this framework on specific systems.

## Contributing new systems or benchmarks

Feel free to add new benchmark apps or support new systems that are part of the
ExCALIBUR benchmarking collaboration.  Read [`CONTRIBUTING.md`](./CONTRIBUTING.md) for more details.
```